homework2/reviewapp/src/app/components/features/shows/ShowReviewSection/ShowReviewSection.tsx

```diff
@@ -0,0 +1,28 @@
+
+import { IReview, IReviewList } from "@/app/typings/review";
+import { IReviewFormProps, ReviewForm } from "../ReviewForm/ReviewForm";
+import { Fragment } from "react";
+import { Heading } from "@chakra-ui/react";
+import { ReviewList } from "../../review/ReviewList/ReviewList";
+
+
+
+export interface ShowReviewSectionProps{
+   reviewList:IReviewList;
+   addReview:(review:IReview)=>void;
+   deleteReview:(review:IReview)=>void;
+
+}
+export const ShowReviewSection=({reviewList,addReview,deleteReview}:ShowReviewSectionProps)=>{
+    return(
+        <Fragment>
```

Comment:
Good example  of when we would use Fragment elements :)

Reply:
Thank you :D